src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -250,21 +261,42 @@ def __init_basic_command_line_for_exec_file(self, test_command_line: list):
                     buf = file_utils.prepare_filelist(input_path, ["*.xml"])
                     self._conformance_ir_filelists.append(buf)
                     buf += ","
-                argument = buf 
+                argument = buf
             else:
                 is_input_folder = False
                 command += f" "
             command += f"{argument}"
         return command
 
+    @staticmethod
+    def __get_suite_filter(test_filter: str, suite_filter: str):
+        filters = test_filter.split(':')
+        suite_filter_mixed = ''
+        for filter in filters:
+            patterns = filter.strip('\"').split('*')
+            suite_filter_part = f'{suite_filter}*'
+            for pattern in patterns:
+                if (pattern and suite_filter.find(pattern) == -1):
+                    suite_filter_part += f'{pattern}*'
+            if suite_filter_part == f'{suite_filter}*':
```

Comment:
Can we improve the condition? Not clear..

Reply:
added additional variable